tests/migration/version_switch_upgrade_target.pm

```diff
@@ -50,11 +50,7 @@ sub run {
     }
 
     record_info('Version', 'VERSION=' . get_var('VERSION'));
-    if (is_pvm) {
-        reconnect_mgmt_console;
-    } else {
-        reset_consoles_tty;
-    }
+    reconnect_mgmt_console if is_pvm;
```

Comment:
Why don't need reset console here?

Reply:
power_action('reboot', textmode => 1, keepconsole => 1);
I though it will keep console, currently passed on x86, s390 and aarch64 and will check on ppc64le

Reply:
https://openqa.nue.suse.com/tests/7097511  PASSED on ppc64le
https://openqa.nue.suse.com/tests/7088862 PASSED on ppc64le